inc/saiport.h

```diff
@@ -1434,6 +1445,15 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_PRIORITY_FLOW_CONTROL_TX,
 
+    /**
+     * @brief Forward or not forward the PFC frame
```

Comment:
When not forwarded, I suppose the PFC frames are honored by the switch queues.  Perhaps it is better to make that clear.